frontend-new/src/App.res

```diff
@@ -93,11 +85,14 @@ let make = () => {
 
   // Fetch benchmarks data
   let ({ReasonUrql.Hooks.response: response}, _) = {
-    let startDate = (Js.Date.getTime(startDate) /. 1000.0)->Js.Json.number
-    let endDate = (Js.Date.getTime(endDate) /. 1000.0)->Js.Json.number
+    let startDate = Js.Date.toISOString(startDate)->Js.Json.string
+    let endDate = Js.Date.toISOString(endDate)->Js.Json.string
```

Comment:
I'm definitely not good at handling dates 😅 Is that the correct format?

Reply:
In this case we're to follow the instructions for Hasura-Postgresql dates: https://hasura.io/blog/postgres-date-time-data-types-on-graphql-fd926e86ee87/

But, yes. ISO8601 is what we need here.